### PR TITLE
Sprint report 34

### PR DESCRIPTION
**Pull request by eduardoj**

### CODE REVIEW

File: _posts/development/2018-03-22-sprint-report-34.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+## Handle Azure credentials in a secure way
+
+Do you know we are working in the Azure Cloud Upload feature of OBS?
+In the next sprints we will include it on our codebase.
+In this one we make sure that the [Azure credentials are handled in a secure way](https://github.com/openSUSE/open-build-service/pull/4644), not showing them in the Webui interface, and encrypting them in the database.
+Stay tuned! :wink:
+
+## Tasks page split off the user page
+So far the information about users and the things they need to work on (submit requests, request review, maintenance updates etc.)
+have been on one page. But account info and involved projects/packages are fairly static info and all the things collaboration are ever
```
Comment (Ana06) [inline _posts/development/2018-03-22-sprint-report-34.md]: why are you breaking the lines so strange? :confused: Do not need to change, but... it is difficult to review it.

File: _posts/development/2018-03-22-sprint-report-34.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+changing. So we decided to give each of them a dedicated page to provide better focus.
+
+![What's splitted](https://trello-attachments.s3.amazonaws.com/57e2914c464225376896419b/59e4b422c24fd7d6b369b351/06dd34cff4c40f3328c993d6fe1878b6/Screenshot-2018-3-5_Home_of_hennevogel_-_openSUSE_Build_Service.png)
+
+
+# Bug fixes
+
+## RabbitMQ Connection Recovery
+
+[Adrian](https://github.com/adrianschroeter) [reported](https://github.com/openSUSE/open-build-service/issues/4522) that after creating a comment led to an 500 error.
```
Comment (Ana06) [inline _posts/development/2018-03-22-sprint-report-34.md]: why is important who reported?

File: _posts/development/2018-03-22-sprint-report-34.md (hunk 1 above)
Comment (eduardoj) [reply _posts/development/2018-03-22-sprint-report-34.md]: Just copy and pasted from etherpad. I din't realize. :hankey:

File: _posts/development/2018-03-22-sprint-report-34.md (hunk 2 above)
Comment (eduardoj) [reply _posts/development/2018-03-22-sprint-report-34.md]: Removed. :hankey: :hankey:

**Comment by eduardoj** [comment]
@Ana06 

> What a short sprint report! :scream: what about mentioning some of the cards we worked on but that we didn't finished? For example, that we refactored the webui controller, although there is still work to do. :thinking:

I just copied and pasted from the etherpad. I think it is enough like it is. I'm also open to more additions.

**Comment by Ana06** [comment]
@eduardoj 

> I just copied and pasted from the etherpad. I think it is enough like it is. I'm also open to more additions.

Then add what we did and was not finished :joy: I can come up with the webui controller, there was a lot of work there, although we didn't finish. I think it is worthwhile to mention it. You worked on that card, so should be easy for you to add it. :wink:

**Comment by eduardoj** [comment]
@Ana06 

> No...why should I? :confused: This is your PR and the webui controller was also your card. :confounded:

Ok. Added.

> If you just copied from the Etherpad (thing that is BTW not an excuse, as if you take care of something, then you should do it properly), why are you the only author of the commit? It would be fair that you make use of severals author Github feature to add the people you copied from. :wink:

Yes, that's fair. Co-authors added. :+1:

Please re-review.

**Pull request merged**